blueocean-plugin/src/main/resources/archetype-resources/src/main/java/Computers.java

```diff
@@ -0,0 +1,163 @@
+package $package;
+
+import hudson.Extension;
+import hudson.model.AbstractBuild;
+import hudson.model.Computer;
+import hudson.model.Executor;
+import hudson.model.Item;
+import hudson.model.Queue;
+import hudson.model.Run;
+import hudson.model.queue.SubTask;
+import io.jenkins.blueocean.rest.OrganizationRoute;
+import io.jenkins.blueocean.rest.factory.BluePipelineFactory;
+import io.jenkins.blueocean.rest.factory.BlueRunFactory;
+import io.jenkins.blueocean.rest.hal.Link;
+import io.jenkins.blueocean.rest.model.BlueOrganization;
+import io.jenkins.blueocean.rest.model.BluePipeline;
+import io.jenkins.blueocean.rest.model.BlueRun;
+import io.jenkins.blueocean.rest.model.Container;
+import io.jenkins.blueocean.rest.model.Resource;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.jenkinsci.plugins.workflow.support.steps.ExecutorStepExecution;
+import org.kohsuke.stapler.Ancestor;
+import org.kohsuke.stapler.Stapler;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+/**
+ * @author kzantow
```

Comment:
This would the right place to put explanatory Javadoc. What are you demonstrating? What might a user want to change?